app/components/sessions/forms/new-session.js

```diff
@@ -35,7 +35,7 @@ export default Component.extend({
       }).then((meetings) => {
       if (meetings.length) {
         const meetingsFromThisYear = meetings.filter((meeting) => meeting.plannedStart && meeting.plannedStart.getFullYear() === this.currentYear);
-        const meetingIds = meetingsFromThisYear.map((meeting) => meeting.number);
+        const meetingIds = meetingsFromThisYear.map((meeting) => meeting.number).filter((meetingId) => meetingId !== undefined);
```

Comment:
@MikiDi @ValenberghsSven Ik heb binnen deze code ook nog even vlug een extra filter toegevoegd voor het volgende: indien er per ongeluk (wat het geval is lokaal en op dev) `undefined` values binnen de meeting id's komen, breakt de berekening voor het aanmaken van een nieuwe id en krijg je uiteindelijk nog meer `undefined` values. Indien dit overbodig lijkt, dan verwijder ik die wel weer.